contracts/utils/AssociatedArrayLib.sol

```diff
@@ -0,0 +1,276 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+library AssociatedArrayLib {
```

Comment:
no library imports for these?

Reply:
Not yet
It's from Smart Sessions
But I don't want to install it as dependency yet 
Can change it later when SS is released (in few weeks)
Or even we can release those libraries as separate repo as they will be needed for many 4337 modules

Reply:
imported it as dep